app/views/songs/index.html.erb

```diff
@@ -0,0 +1,19 @@
+<div class="headline">
+  <h1><%= title(t('.title')) %></h1>
+</div>
+
+<div class="col-md-12">
+  <%= grid(@songs) do |g|
+    g.column(name: t('.name'), attribute: 'title') do |song|
```

Comment:
You can change this to: `Song.human_attribute_name(:title)` and remove `sv.songs.index.name` from the treanslation file.